KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -870,6 +870,8 @@
 
 "kh_uisdk_quotes_error_missing_addresses_subtitle" = "Please enter a pickup and drop off address to see quotes";
 
+"kh_uisdk_error_incorrect_sdk_version_message" = "Please update your application before using this feature";
```

Comment:
This needs to be localised. Is this key in Phrase?

Reply:
<img width="799" alt="Screenshot 2023-04-18 at 16 17 04" src="https://user-images.githubusercontent.com/97114085/232805703-1aa27e57-8b9b-453e-b832-b8c6fcc3cabb.png">
Yes